app/src/main/res/layout/profile_input_view.xml

```diff
@@ -5,7 +5,6 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_marginStart="28dp"
-    android:layout_marginTop="28dp"
```

Comment:
For now I would suggest not to make any change `profile_input_view` as it will bring change to all other views as it is used in various files. Instead modify your xml to match the closest mock padding/margin and this will be fixed as a part of mini project.

Reply:
It cannot be adjusted as  margin is lesser than 28.

Reply:
As discussed, you can just remove the margin from all sides and manage that in parent layout so that while working on other UIs it becomes easier.

Reply:
Done.